tracr/io/read.py

```diff
@@ -1,11 +1,14 @@
 # tries to guess the format of the file
 from .util import guess_format, check_format
 from .tif import read as read_tif
+from .dicom import read as read_dicom
 
 def read(filename, **kwds):
     fmt = kwds.get('format', guess_format(filename))
     fmt = check_format(fmt)
     if fmt == 'tif':
```

Comment:
Make sure that `check_format` knows how to handle the various permutations of dicom file extensions, e.g. "dcm", "DCM", "dicom", "DICOM", etc.